kernel.spec.in

```diff
@@ -118,8 +120,10 @@ Source33:       check-for-config-changes
 Source34:       gen-config
 Source100:      config-base
 Source101:      config-qubes
+%if 0%{?fedora} >= 32
```

Comment:
Don't exclude sources, src.rpm should be the same regardless of where you build it.